server/pulp/server/webservices/views/plugins.py

```diff
@@ -9,7 +9,33 @@
 
 
 class DistributorResourceView(View):
-    pass
+    """
+    Views for a single distributor.
+    """
+
+    @auth_required(authorization.READ)
+    def get(self, request, distributor_id):
+        """
+        Return a serialized object representing the specified distributor.
+
+        :param request       : WSGI request object
+        :type  request       : django.core.handlers.wsgi.WSGIRequest
+        :param distributor_id: id of distributor to match
+        :type  distributor_id: string
+        :return              : Serialized object representing the specified distributor
```

Comment:
Getting picky about semantics, I don't think the object you're returning has been serialized. It is a response object that contains serialized data.